code/modules/projectiles/projectile/bullettypes.dm

```diff
@@ -360,3 +360,8 @@ There are important things regarding this file:
     if(istype(target))
         var/obj/item/ammo_casing/crossbow/bolt/R = new(null)
         target.embed(R, def_zone)
+
+/obj/item/projectile/bullet/bolt/mech
```

Comment:
Depending on how fast this fires and how many shots this holds, this could be overpowered. It doesn't exactly have recoil, and deals 68 damage each hit. It has a cooldown of only 1 second too, right?